buf-kvutils.yaml

```diff
@@ -3,30 +3,27 @@
 
 version: v1beta1
 
-deps:
-  - buf.build/googleapis/googleapis
-
 build:
   roots:
-    - daml-lf/archive/src/main/protobuf/
-    - daml-lf/transaction/src/main/protobuf/
-    - ledger/ledger-configuration/protobuf/
-    - ledger/participant-state/kvutils/src/main/protobuf/
-    - ledger-api/grpc-definitions
+    - daml-lf/archive/src/main/protobuf
+    - daml-lf/transaction/src/main/protobuf
+    - ledger/ledger-configuration/protobuf
+    - ledger/participant-state/kvutils/src/main/protobuf
   excludes:
-    # We have to exclude these to avoid duplicate definitions of e.g. Unit within the same package.
+    # We have to exclude these to avoid duplicate definitions of e.g. Unit
+    # within the same package.
     - daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_11/
     - daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_12/
     - daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_13/
     - daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_14/
     - daml-lf/archive/src/main/protobuf/com/digitalasset/
 
-lint:
-  use:
-    - DEFAULT
-  except:
-    - PACKAGE_VERSION_SUFFIX
-
 breaking:
   use:
-    - WIRE_JSON
+    # WIRE is enough here as KVUtils data is internal and will be transmitted
+    # and stored in binary protobuf format for the foreseeable future.
+    - WIRE
+  except:
+    # Moving an individual field into a new `one_of` retains binary backwards
+    # compatibility.
+    - FIELD_SAME_ONEOF
```

Comment:
Removing this check also allows for incompatible changes but we have to use it at least until 1.17.0 as there are no finer-grained checks for backwards-compatible cases.

Reply:
ack.